Lib/logging/__init__.py

```diff
@@ -883,25 +891,20 @@ def _removeHandlerRef(wr):
     # set to None. It can also be called from another thread. So we need to
     # pre-emptively grab the necessary globals and check if they're None,
     # to prevent race conditions and failures during interpreter shutdown.
-    acquire, release, handlers = _acquireLock, _releaseLock, _handlerList
-    if acquire and release and handlers:
-        acquire()
-        try:
-            handlers.remove(wr)
-        except ValueError:
-            pass
-        finally:
-            release()
+    acquire, handlers = _get_lock, _handlerList
+    if acquire and handlers:
+        with acquire():
```

Comment:
Why renaming _get_lock to acquire? It's misleading.

Reply:
Agree with Victor.

Reply:
Updated